http/swagger.yml

```diff
@@ -8840,9 +8840,6 @@ components:
         flux:
           description: The Flux script to run for this task.
           type: string
-        token:
-          description: The token to use for authenticating this task when it executes queries.
-          type: string
       required: [flux, token]
```

Comment:
does it need to be removed as well?

Reply:
I asked @jademcgough if we should remove it or deprecate we decided to just remove it.

Reply:
I mean i think u also want to remove the token from the required, right

Reply:
oo your right Thanks!